src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/library.cljs

```diff
@@ -137,15 +171,27 @@
                            (if success?
                              (do (modal/pop-modal)
                                  ((:request-refresh props)))
-                             (swap! state assoc :server-error (get-parsed-response false))))})))))})
-
+                             (swap! state assoc :server-error (get-parsed-response false))))})))))
+   :component-did-mount
+   (fn [{:keys [props state refs]}]
+     (let
+       [attributes (get-in props [:workspace :workspace :library-attributes])
+        properties (get-in props [:library-schema :properties])]
+       (doseq [[property-key {:keys [typeahead] :as property}] properties]
+                (if (= typeahead "ontology")
+                  (let [pk-str (name property-key)]
+                    (select2-typeahead {:id  (get attributes :library:diseaseOntologyDOID -1 )
+                                        :label (get attributes :library:diseaseOntology "Select a disease ontology.")}
+                                       (@refs pk-str)
+                                       (fn [id label]
+                                         (swap! state assoc :ontologyText label :ontologyDOID id))))))))})
```

Comment:
add a TODO comment: when we support multiple ontology-driven properties, can't hardcode the keys in state or attributes